### DISCUSSION TITLE
[AVR Miner] and [PC Miner] Rejected (Username changed) but didn't change username

### DISCUSSION THREAD

**Discussion by Darksplat** [revoxhere/duino-coin#1597] [Q&A]
Hi I keep on getting the Rejected (Username changed) error on my setup. I haven't changed my physical setup in months and just around Christmas I started to get this error. 

My setup is this

2 PICO's
9 UNO/Megas Connected through a 10 port hub
1 RPI3
1 ESP32 - connected to the UPS
1 EPS8622 - connected to the UPS

All devices have the latest software 3.33, the ESP's are happy running by themselves its just the uno's and and the RPI that are giving me trouble, I am running the latest RPI OS and it is updated running python 3.9.xxx 

I have tried just running the RPI by itself and the UNO/Mega and the PICO's all by themselves with all the same error

any ideas?

cheers

**Comment by Grantrocks** [comment]
Try updating the config files
